InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -14612,7 +14618,8 @@ private string GetNotecardSegment(SceneObjectPart part, string name, int line, i
             {
                 AsyncCommands.
                 DataserverPlugin.DataserverReply(reqIdentifier, NotecardCache.GetLine(assetID, line, startOffset, maxLength));
-                ScriptSleep(DELAY);
+                if (((line % LINES_PER_DELAY) == 0) && (startOffset == 0))
```

Comment:
While clever and stateless, this runs afoul of an attack vector that trivially defeats it: simply have your script only work with lines that have no delay.  _Et voila_ the whole purpose of this has been nullified.